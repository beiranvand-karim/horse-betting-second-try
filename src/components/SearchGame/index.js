import React from "react"

export const SearchGame = (props) => {
   return <form action="" onSubmit={props.searchGameFormSubmit}>
      <input id="gamedId" name="gamedId" onChange={props.change} value={props.gameId} type="text"/>
      <button>search</button>
   </form>
};
export default SearchGame