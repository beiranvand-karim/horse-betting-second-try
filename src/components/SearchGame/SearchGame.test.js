/* global describe, it, expect*/
import React from "react"
import SearchGame from "./"
import {MemoryRouter} from "react-router-dom"
import renderer from "react-test-renderer"
import {shallow} from "enzyme"

describe('<SearchGame/>', () => {
   it('should render without crashing', () => {
      shallow(<SearchGame/>)
   });

   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><SearchGame/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});