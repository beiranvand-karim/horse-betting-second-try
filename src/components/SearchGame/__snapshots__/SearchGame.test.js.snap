// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchGame/> should matches the snapshot 1`] = `
<form
  action=""
>
  <input
    id="gamedId"
    name="gamedId"
    type="text"
  />
  <button>
    search
  </button>
</form>
`;
